cmd/docsite/site.go

```diff
@@ -184,12 +194,21 @@ func openDocsiteFromConfig(configData []byte, baseDir string) (*docsite.Site, *d
 		}
 		return http.Dir(filepath.Join(baseDir, dir))
 	}
-	site.Templates = httpDirOrNil(config.Templates)
-	site.Content = nonVersionedFileSystem{httpDirOrNil(config.Content)}
-	site.Assets = httpDirOrNil(config.Assets)
+
+	if DEBUG {
+		content := newVersionedFileSystemURL(CODEHOST_URL, "master")
+		if _, err := content.OpenVersion(context.Background(), ""); err != nil {
+			return nil, nil, errors.WithMessage(err, "downloading content default version")
+		}
+		site.Content = content
+	} else {
```

Comment:
Necessary to keep?

Reply:
I want to keep it to make it easier to debug in the future since I'm probably the one to fix things anyway.